pkgs/development/python-modules/django-extensions/default.nix

```
@@ -16,6 +16,13 @@ buildPythonPackage rec {
     sha256 = "0053yqq4vq3mwy7zkfs5vfm3g8j9sfy3vrc6xby83qlj9wz43ipi";
   };
 
+  patches = [
```

do you mind adding a comment so it's easy to review is this is still applicable in future updates
```suggestion
  patches = [
    # fixes issue with ...., should be able to be removed in next release
```